packages/client/lib/blockchain/chain.ts

```diff
@@ -110,7 +110,7 @@ export class Chain extends EventEmitter {
         db: options.chainDB,
         common: this.config.chainCommon,
         validateBlocks: true,
-        validateConsensus: false,
+        validateConsensus: true,
```

Comment:
This is putting too much load on the PoW sync, I guess what we can do is just to activate for PoA by including a conditional switch along the `common` instance setting from above.